docs/seeding.md

```diff
@@ -21,7 +21,7 @@ Seeders use the [EDGI Archiving Primers](https://envirodatagov.org/archiving/),
 
 ### Crawlable URLs
 
-- URLs judged to be crawlable are "nominated" (equivalently, "seeded") to the End of Term project (EOT), using the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en).
+- URLs judged to be crawlable are "nominated" (equivalently, "seeded") to the Internet Archive, using the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en).
```

Comment:
Awesome, again tho to be consistent with language I think we want something like "save URLS to the Internet Archive"